MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -1623,21 +1635,8 @@ public void refitUnit(Refit r, boolean selectModelName) {
                 if (getCampaign().isWorkingOnRefit(tech) || tech.isEngineer()) {
                     continue;
                 }
-                StringBuilder nameBuilder = new StringBuilder(128);
-                nameBuilder.append("<html>")
-                    .append(tech.getFullName())
-                    .append(", <b>")
-                    .append(SkillType.getColoredExperienceLevelName(tech.getSkillLevel(getCampaign(), false)))
-                    .append("</b> ")
-                    .append(tech.getPrimaryRoleDesc())
-                    .append(" (")
-                    .append(getCampaign().getTargetFor(r, tech).getValueAsString())
-                    .append("+), ")
-                    .append(tech.getMinutesLeft())
-                    .append('/')
-                    .append(tech.getDailyAvailableTechTime())
-                    .append(" minutes</html>");
-                name = nameBuilder.toString();
+                String nameBuilder = "<html>" + tech.getFullName() + ", <b>" + SkillType.getColoredExperienceLevelName(tech.getSkillLevel(getCampaign(), false)) + "</b> " + tech.getPrimaryRoleDesc() + " (" + getCampaign().getTargetFor(r, tech).getValueAsString() + "+), " + tech.getMinutesLeft() + '/' + tech.getDailyAvailableTechTime() + " minutes</html>";
```

Comment:
This seems less clean, plus if we're not using a StringBuilder you can just concatenate everything into `name` and skip the second line.

Was this an IDE-suggested change?  Apparently most will turn this form into the StringBuilder form under the hood, so I'm not sure this change gets us anything.